pos_sequence_ref_number/models/pos_order.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# © 2016 Acsone SA/NV (http://www.acsone.eu)
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, models
+
+
+class PosOrder(models.Model):
+    _inherit = "pos.order"
+
+    @api.model
+    def sequence_number_sync(self, vals):
+        next = vals.get('_sequence_ref_number', False)
+        next = int(next) if next else False
+        if vals.get('session_id') and next is not False:
+            session = self.env['pos.session'].browse(vals['session_id'])
+            if next != session.config_id.sequence_id.number_next_actual:
```

Comment:
Updating the ir.sequence is a problem because only the POS Manager is allowed to do it by ACL's.
You should add once for POS User